doc/process/jenkins_job_configuation_access.md

```diff
@@ -0,0 +1,90 @@
+# Introduction
+
+There are a number of cases were we would like working group members
+to be able to run and or edit jobs within jenkins. Examples include
+the CITGM jobs and the Benchmarking jobs.
+
+By default jobs can only be edited by build working group members
+and start/stopped etc. by collaborators. We do not give access to
+everbody to run/edit as it would be a potential security issue,
+both through the potential disclosure of secrets used as
+part of the builds, as well as a potential denial
+of service if jobs are launched maliciously.
+
+This document outlines the processes by which we will allow
+people outside the build working group to edit jobs as well
+as allowing people who are not collaborators run specific jobs.
+
+Unfortunately jenkins does not allow us to easily delegate the
+creation of new jobs.  As such the creation of a new job will
+need to be requested through an issue in the build repo.  Any
+build member can then create a blank job in the group for the
+working group which can then be edited by those with access
+to the jobs for that working group.
+
+## Ongoing access
+
+In the first case we want to be able to provide ongoing access to
+an individual to edit or run a job associated with a working group
+when this supports the efforts of a working group/team. For
+example, some members of the benchmarking team will be regularly
+modifying benchmarking jobs and therefore the ability to configure
+benchmarking jobs needs to be granted in an ongoing manner.
+
+In these cases the following will be used to decide if ongoing access
+can be provided:
```

Comment:
@gibfahn yeah; that's pretty much my thoughts as well. The possible distinction would be that each team can have an 'admin' group if the team is big so not all have free reign.